### DISCUSSION TITLE
华中科技大学（主校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#52] [University]
> 来自群友

**宿舍是上床下桌吗**
是

**教室和宿舍有没有空调**
均有

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
有独立卫浴

**有早自习、晚自习吗**
没有（不排除有些院系自行组织）

**有晨跑吗**
没有

**每学期跑步打卡的要求是多少公里，可以骑车吗**
32 次，女生 2km，男生 3km，可以骑车

**寒暑假放多久，每年小学期有多长**
大概 100 天？没有小学期

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许点，一部分外卖可以送到楼下。大部分需要自己骑车去校门拿，骑车大概 5min

**学校交通便利吗，有地铁吗，在市区吗，不在的话进城要多久**
交通便利，学校离地铁站很近，主校区在光谷

**宿舍楼有洗衣机吗**
有，但是很多宿舍会自己购置洗衣机

**校园网怎么样**
慢大部分时候还可以叭

**每天断电断网吗，几点开始断**
十二点以后断网（周五周六不断网）

**食堂价格贵吗，会吃出异物吗**
比较便宜，食堂很棒！

**洗澡热水供应时间**
中午 11：30 — 14：00，晚上 18：00 — 24：00

**校园内可以骑电瓶车吗，电池在哪能充电**
可以，在宿舍楼下充电

**宿舍限电情况**
大功率的吹风机这些不可以用

**通宵自习有去处吗**
有，但是不多

**大一能带电脑吗**
可以

**学校里面用什么卡，饭堂怎样消费**
校园卡

**学校会给学生发银行卡吗**
会

**学校的超市怎么样**
挺多的，价格也比较合理，基本可以满足生活需要

**学校的收发快递政策怎么样**
在快递点收取

**学校里面的共享单车数目与种类如何**
数目多，电动车和自行车都有

**现阶段学校的门禁情况如何**
对于学生来说还好，校外人员不可以随便进入，需要提前预约

**宿舍晚上查寝吗，封寝吗，晚归能回去吗**
不查寝，不封寝

**Comment by NMDWSM0** [comment]
补充一下，11点断电
